### DISCUSSION TITLE
[1277930]Riddle Joker

### DISCUSSION THREAD

**Discussion by Onism02** [pjy612/SteamManifestCache#623] [Share]
### Steam 游戏ID

1277930

### 密钥列表

```hash
1277931;b3f09329b6a21d00c30e4eb199d906ced33932b7f75e2528e33e1ce41c3c4d7b
```

### 清单附件

[[1277930]Riddle Joker.zip](https://github.com/pjy612/SteamManifestCache/files/15139273/1277930.Riddle.Joker.zip)

### 备注

code:1A4DF03F2CB6AAC2
email:nujabes@88.com

**Comment by zhuifengzhe1** [comment]
清单不可以用了

**Reply by pjy612** [reply]
好像是服务器抽风
